main.cpp

```diff
@@ -52,85 +52,94 @@ arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
+//  Noun: musicPlayer
+//  action 1: play music
+musicPlayer.play();
+//  action 2: stop music
+musicPlayer.stop();
+//  action 3: skip currently playing song
+musicPlayer.skipSong();
 
-//  action 2:
-
-//  action 3:
- 
 //  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
+//  Noun: phone
+//  action 1: read numbers inputted
+phone.readInput();
+//  action 2: connect to network
+phone.connectNetwork();
+//  action 3: transmit voice signal from the people talking
+phone.transmitSignal();
 
-//  action 3:
- 
 //  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun: amplifier
+//  action 1: take audio signal input
+amplifier.takeInput();
+//  action 2: amplifie signal
+amplifier.amplify();
+//  action 3: make noise
+amplifier.makeNoise();
  
 //  4)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun: riceCooker
+//  action 1: cook rice
+riceCooker.cook();
+//  action 2: store rice
+riceCooker.store();
+//  action 3: reheat other food
+riceCooker.reheat(); 
  
 //  5)
-//  Noun:
-//  action 1:
-
-//  action 2:
+//  Noun: fridge
+//  action 1: cool things
+fridge.cooling();
+//  action 2: preserve ingredients
+fridge.preserve();
+//  action 3: make ice
+fridge.makeIce();
 
-//  action 3:
- 
 //  6)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun: cow
+//  action 1: make milk
+cow.makeMilk();
+//  action 2: say moo
+cow.sayMoo();
+//  action 3: eat grass
+cow.eatGrass();
  
 //  7)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun: bird
+//  action 1: fly around
+bird.flyAround();
+//  action 2: drop poop
+bird.dropPoop();
+//  action 3: chirp
+bird.chirp();

```

Comment:
As explained in the instructions, here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
**REPLACE** the highlighted noun/action as follows:
- [ ] come up with a new noun and 3 actions
- [ ] pick a noun that needs `camelCase` for the name
- [ ] pick actions that need `camelCase` for their name.

**Do not** create an 11th noun.